src/function/table/call/show_attached_databases.cpp

```diff
@@ -0,0 +1,67 @@
+#include "function/table/call_functions.h"
+#include "main/database_manager.h"
+
+using namespace kuzu::common;
+using namespace kuzu::catalog;
+
+namespace kuzu {
+namespace function {
+
+struct ShowAttachedDatabasesBindData : public CallTableFuncBindData {
+    std::vector<main::AttachedDatabase*> attachedDatabases;
+
+    ShowAttachedDatabasesBindData(std::vector<main::AttachedDatabase*> attachedDatabases,
+        std::vector<LogicalType> returnTypes, std::vector<std::string> returnColumnNames,
+        offset_t maxOffset)
+        : CallTableFuncBindData{std::move(returnTypes), std::move(returnColumnNames), maxOffset},
+          attachedDatabases{std::move(attachedDatabases)} {}
+
+    std::unique_ptr<TableFuncBindData> copy() const override {
+        return std::make_unique<ShowAttachedDatabasesBindData>(attachedDatabases, columnTypes,
+            columnNames, maxOffset);
+    }
+};
+
+static common::offset_t tableFunc(TableFuncInput& input, TableFuncOutput& output) {
+    auto& dataChunk = output.dataChunk;
+    auto sharedState =
+        ku_dynamic_cast<TableFuncSharedState*, CallFuncSharedState*>(input.sharedState);
+    auto morsel = sharedState->getMorsel();
+    if (!morsel.hasMoreToOutput()) {
+        return 0;
+    }
+    auto& attachedDatabases =
+        ku_dynamic_cast<function::TableFuncBindData*, ShowAttachedDatabasesBindData*>(
+            input.bindData)
+            ->attachedDatabases;
+    auto numDatabasesToOutput = morsel.endOffset - morsel.startOffset;
+    for (auto i = 0u; i < numDatabasesToOutput; i++) {
+        auto attachedDatabase = attachedDatabases[morsel.startOffset + i];
+        dataChunk.getValueVector(0)->setValue(i, attachedDatabase->getDBName());
+        dataChunk.getValueVector(1)->setValue(i, attachedDatabase->getDBType());
+    }
+    return numDatabasesToOutput;
+}
+
+static std::unique_ptr<TableFuncBindData> bindFunc(main::ClientContext* context,
+    TableFuncBindInput*) {
+    std::vector<std::string> columnNames;
+    std::vector<LogicalType> columnTypes;
+    columnNames.emplace_back("name");
```

Comment:
I forgot what our conventions are. Should we capitalize the first letter?

Reply:
I don't think we capitalize any letters in show_tables()